lib/jpegstream.js

```diff
@@ -10,7 +10,8 @@
  * Module dependencies.
  */
 
-var Stream = require('stream').Stream;
+var Readable = require('stream').Readable;
```

Comment:
(in browserify the stream module is based on readable-stream so it's no big deal)